lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -43,19 +74,69 @@ class MutationExtensionSpec<T : Any> : ExtensionSpec<T> {
   }
 
   /**
-   * Register a callback that will be invoked when an item is removed by swiping it
+   * Allows to define a [RowSpec] (usual `create/bind` mechanism) for a view that will be drawn
+   * below a swiped row. It's created when the swipe is initiated and bind is called with every
+   * movement receiving [SwipeBindData] so:
+   * - the original swiped view can be retouched accorindg to the percentage (alpha for instance)
+   * - the original dataItem, in case its data is needed for the underview, and the underview itself.
+   * WARNING: this API might evolve as currently:
```

Comment:
Can you confirm this formats correctly? I think this warning will be considered part of the previous like and the subsequent bullet points will be indented at the same level, which doesn't seem to be what you want.